Doc/library/functools.rst

```diff
@@ -664,6 +669,43 @@ The :mod:`functools` module defines the following functions:
    would have been ``'wrapper'``, and the docstring of the original :func:`example`
    would have been lost.
 
+.. function:: get_variants(key)
+
+   Return all registered function variants for this key. Function variants are
+   objects that represent some subset of the functionality of a function, for
```

Comment:
Not too happy with this wording. @AlexWaygood do you have suggestions?

Reply:
At the very least describe the return type better. Is it a list? What are the items?